CIP-dapp-bridge-rework/README.md

```diff
@@ -0,0 +1,1169 @@
+
+At the same time, we are not required to use low-level libraries to solve simple tasks. It's sufficient to just generate JSONs.
+
+For more complex tasks related to working with smart contracts, the DApp should use more low-level solutions for generating redeemers and dataHashes. However, this will be part of the DApp's direct logic, necessary for its operation, rather than redundant logic that mimics a full-fledged wallet with coinSelection algorithms, etc. In other words, the complexity of the code will simply correspond to the complexity of the task.
+
+## Rationale: how does this CIP achieve its goals?
+
+This CIP addresses the issues described in the `Motivation` section:
+
+1. **Complexity in Writing Client-Side Logic for DApps**: The minimal amount of code required for DApp to interact with the wallet has been dramatically reduced.<br><br>
+
+2. **Integration Difficulties**: The interaction between DApp and wallet is now easily mapped onto the logic of the wallet's interaction with the end user. Declarative rather than imperative.<br><br>
+
+3. **Privacy Issues**: The wallet is no longer obliged to disclose all information and decides for itself what data can be published and what cannot. Now it is responsible for managing UTxO and other crucial aspects, as it should be.<br><br>
+
+4. **Current Role of Wallets**: The wallet now acts as a wallet, not a "submit endpoint," and the DApp acts as its client, not a substitute.<br><br>
+
+5. **Scalable Complexity**: The complexity of the code on the DApp side now precisely matches the complexity of the tasks it solves. Nothing superfluous.<br><br>
+
+6. **Reducing Dependencies**: DApp developers are no longer required to use libraries for working with Cardano CDDL to interact with the wallet, if there is no need for them. The pipeline has become smooth and sequential, reducing the number of failure points.<br><br>
+
+7. **Lack of abstraction**: DApps no longer need to adapt to every low-level change in Cardano. They now have an **abstraction layer** that solves this problem once and for all.<br><br>
+
+8. **Security**: The creation of transactions is now fully controlled by the user's wallet and does not leave its confines. Users can be confident that third parties are not involved in managing their funds.<br><br>
+
+9. **Trustful system**: Users no longer have to trust DApps and can safely connect their wallets to new applications without the risk of misuse on their part.<br><br>
+
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [] The interface is implemented and supported by various wallet providers.
+- [] The interface is used by DApps to interact with wallet providers.
+
+### Implementation Plan
+
+- [] Provide some reference implementation of wallet providers
+- [] Provide some reference implementation of the dapp connector
```

Comment:
```suggestion
- [ ] The interface is implemented and supported by various wallet providers.
- [ ] The interface is used by DApps to interact with wallet providers.

### Implementation Plan

- [ ] Provide some reference implementation of wallet providers
- [ ] Provide some reference implementation of the dapp connector
```
addendum to last format-nitpick review: these only format properly as tick-boxes on GitHub & other Markdown previewers when either a space or `x` (maybe other characters?) is between the brackets.